daemon/container_operations_unix.go

```diff
@@ -718,6 +722,11 @@ func (daemon *Daemon) DisconnectFromNetwork(container *container.Container, n li
 	if err := container.ToDiskLocking(); err != nil {
 		return fmt.Errorf("Error saving container to disk: %v", err)
 	}
+
+	attributes := map[string]string{
+		"container": container.ID,
+	}
+	daemon.LogNetworkEventWithAttributes(n, "disconnect", attributes)
 	return nil
 }
 
```

Comment:
@calavera Actually the `releaseNetwork()` is called by the monitor process every time the container stops.
And also on container start failure, but only if the network connect was successful.

In either case the logging would happen only for the networks the container was actually connected to.